pylintrc_backup

```diff
@@ -2,7 +2,7 @@
 # ** DO NOT EDIT THIS FILE **
 # ***************************
 #
-# This file was generated by edx-lint: https://github.com/edx/edx-lint
```

Comment:
Hmm, we should just delete this file, it shouldn't be here. I can take care of that in a separate PR, though.

Reply:
Hmm. In all my test repos this file never showed up, but it did in a number of other repos. I wonder if it's a `.gitignore` thing. I can remove it.

Reply:
Ah, got it. Thanks!